src/WebAssert.php

```diff
@@ -61,12 +67,19 @@ public function addressEquals($page)
      *
      * @throws ExpectationException
      */
-    public function addressNotEquals($page)
+    public function addressNotEquals($page, $timeout = 10)
     {
-        $expected = $this->cleanUrl($page);
-        $actual = $this->getCurrentUrlPath();
+        $cleanedUrl = $this->cleanUrl($page);
+        $actual = null;
 
-        $this->assert($actual !== $expected, sprintf('Current page is "%s", but should not be.', $actual));
+        $this->assert(
+            $this->session->getPage()->waitFor($timeout, function () use ($cleanedUrl, &$actual) {
+                $actual = $this->getCurrentUrlPath();
+                // TODO Don't fail on first try if address are different.
+                // TODO This condition is true and break the loop.
+                return $actual !== $cleanedUrl;
```

Comment:
@lenybernard I think that this condition can't work.

Let's say that the test expect the address to be different from `path/`:
- if the URL is `path/` during one second, then '/file', the test will work (after trying during one second)
- if the URL is `/file` during one second, then 'path/', the test will work, the condition will be evaluated to true during the first loop iteration, it won't wait several seconds, so the test will be marked as passed incorrectly